apps/website/src/module/member/ui/component/member-card/member-card.presenter.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Member } from '@nitic-astronomy/model';
+import type { FC, ComponentPropsWithoutRef } from 'react';
+import { twMerge } from '@/core/util/tw-merge.util';
+import { MemberExternalLink } from '@/module/member/ui/component/member-external-link';
+import { MemberIcon } from '@/module/member/ui/component/member-icon';
+import SquareLightLogoImage from '@public/image/logo/square-light.png';
+
+export type MemberCardProps = ComponentPropsWithoutRef<'li'> & Pick<Member, 'name' | 'icon' | 'role' | 'description' | 'externalLinks'>;
+
+export const MemberCard: FC<MemberCardProps> = ({ name, icon, role, description, className, externalLinks, ...props }) => (
+  <li className={twMerge('flex flex-col items-stretch gap-2 rounded-lg bg-keyplate-1 p-6 shadow-floating', className)} {...props}>
```

Comment:
`MemberCard` からしたらリストかそうじゃないかは関心事じゃない (外部からリストかリストじゃないかを変えられるべき) なので、 `div` でもよくないかとは思う

それか `MemberCardListItem` にして `<li>` であることを命名で強調されてるとわかりやすいかなぁ

あとは props 経由でラッパを注入する方法もあるっちゃある
https://zenn.dev/sho_ts/articles/ff3082b1e3db5f

```suggestion
  <div className={twMerge('flex flex-col items-stretch gap-2 rounded-lg bg-keyplate-1 p-6 shadow-floating', className)} {...props}>
```

Reply:
https://github.com/astronomy-club-at-nitic/nitic-astronomy/pull/166/commits/a3c282abc47f90c1e6aab3d031ab7d1313c5cc3b